index.js

```diff
@@ -2,14 +2,59 @@
 const execa = require('execa');
 const findVersions = require('find-versions');
 
-module.exports = (binary, options = {}) => {
-	return execa(binary, options.args || ['--version'])
-		.then(result => findVersions(result.stdout || result.stderr, {loose: true})[0])
-		.catch(error => {
+const knownBinaryArgs = new Map([
+	['openssl', ['version']],
+	['ffmpeg', ['-version']]
+]);
+
+const getKnownBinaryArgs = binary => {
+	let args = knownBinaryArgs.get(binary);
+	if (args !== undefined) {
+		return args;
+	}
+
+	args = knownBinaryArgs.get(binary.trim());
```

Comment:
Why not just `.trim()` by default?

Reply:
Actually, whole trim part was incorrect. Executables can contain spaces.